src/hooks/react-query/feature-requests/useFeatureRequests.js

```
@@ -0,0 +1,211 @@
+            },
+        ]);
+
+        if (error) {
+            console.error('Error voting feature request:', error);
+            throw new Error('Failed to vote');
+        }
+    }
+};
+
+// Hook to vote/unvote on a feature request
+export const useVoteOnFeatureRequest = (featureRequestId) => {
```

ESLint

[error] 'featureRequestId' is defined but never used.

(no-unused-vars)